test/intfuncs.jl

```diff
@@ -487,3 +487,17 @@ end
     ci = code_typed(() -> 14 // 21)[][1]
     @test ci.code == Any[Core.ReturnNode(2 // 3)]
 end
+@testset "binomial" begin
+    for T in (Int8, UInt16, Int32, UInt64)
+        for x in rand(isqrt(typemax(T)):isqrt(typemax(T)), 1000)
```

Comment:
IIUC, `fill(isqrt(typemax(T)), 1000)` would be a more obvious way to put this, but the intention was probably something else?

Reply:
oops.